src/shared_realm.hpp

```diff
@@ -210,6 +212,7 @@ class Realm : public std::enable_shared_from_this<Realm> {
     void invalidate();
     bool compact();
     void write_copy(StringData path, BinaryData encryption_key);
+    OwnedBinaryData write_copy_to_mem();
```

Comment:
What would you think of it just being `write_copy()`? The overload should make it pretty clear what is going on.